docs/docs/self-host/lightdash-cloud-vs-self-hosted.mdx

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 2
+sidebar_label: Cloud vs. Self-Hosted
+---
+
+# Lightdash Cloud vs. Self-Hosted
+
+Lightdash is open-source and licensed under the MIT license. This means that you are free to self-host Lightdash on 
+your own infrastructure but without any warranty or support from the Lightdash team. In other words, it's up to you 
+to maintain and monitor your Lightdash instance.
+
```

Comment:
This is a great addition! 

I wonder if we can be even more like "harsh" here? As in, being quite explicit about needing to basically be a data engineer if you want to self-host Lightdash. You need to be comfortable with Docker, etc.?

Reply:
Updated this! There is also a similar disclaimer at the top of the main self hosting page too